src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1094,6 +1094,18 @@ else if(jvbCall.getCallState() == CallState.CALL_ENDED)
         properties.put(ProtocolProviderFactory.ENCRYPTION_PROTOCOL_STATUS
             + ".DTLS-SRTP", "true");
 
+        AbstractGateway gw = gatewaySession.getGateway();
+        if(gw instanceof SipGateway)
+        {
+            String preventAuthLogin
+                = ((SipGateway) gw).getSipAccountProperty("PREVENT_AUTH_LOGIN");
+
+            if (Boolean.valueOf(preventAuthLogin))
+            {
+                return properties;
+            }
```

Comment:
Returning the properties here will prevent using any custom overrides coming from org.jitsi.jigasi.xmpp.acc properties, not only the authentication.